docs/install/server-infrastructure.md

```diff
@@ -132,7 +132,7 @@ This shows us that `QmdzvHZ...` is pinned across the three IPFS nodes within our
 1. When you're finished playing around, kill the cluster. You may have to run this as root:
 
     ```shell
-    docker-compose kill
+    sudo docker-compose kill
```

Comment:
`sudo` here seems weird.

This is up to local setups and your user being in the docker group or not.
There is big security implications of giving it out the docker group, and not having it on your main user probably make sense.

However our docs assume you do, many default installs do that.
It's also very easy for users to figure out what is the issue just by googling the error message if they stamble on it.